src/main/clojure/clara/rules/dsl.clj

```diff
@@ -47,7 +47,12 @@
   "Creates a condition with the given optional result binding when parsing a rule."
   [condition result-binding expr-meta]
   (let [type (if (symbol? (first condition))
-               (if-let [resolved (resolve (first condition))]
+               (if-let [resolved (and
+                                  ;; If we are compiling ClojureScript rules we don't want
+                                  ;; to resolve the symbol in the ClojureScript compiler's
+                                  ;; Clojure environment.  See issue 300.
+                                  (not (com/compiling-cljs?))
+                                  (resolve (first condition)))]
```

Comment:
Yeah, I agree it is a bit odd but probably outside the scope of https://github.com/cerner/clara-rules/issues/296 or https://github.com/cerner/clara-rules/issues/300